### PR TITLE
replace blur with dim transition

### LINKED ISSUE #884: Mystery black border flash when dropping down Docs menu
I went to the docs homepage and moved the mouse over the Docs drop-down.  When I did this, the background gradually blurred, and a strange blurry black border started to appear around the edges of the (visible area of the) page.  After a moment the blur reached its maximum blurriness at which point the black border suddenly vanished.

MS Edge on Windows 10.

image

### PR DESCRIPTION
**Pull request by karthik2804**
Fixes #884 

The blur overlay seemed odd and at times slow, so I decided to replace it with a dim animation. 

image

If that seems acceptable, we can merge the PR else I will look at the blur animation.

### CODE REVIEW

**Comment by itowlson** [comment]
The current dim level is maybe a bit dramatic for my taste (e.g. the drop-down now feels isolated and unmoored).  Could we try it with a more gentle dimming?  The user's attention is already in the right part of the screen, we don't need to over-emphasise the focus assist.  Thanks though, the direction seems nice and this is very much a nit!

(by the way, the formatting changes made it a bit laborious to see what the _actual_ CSS change was - it would be good to do those in a separate commit or PR so that someone coming back to this can easily see exactly what change was made)

**Comment by karthik2804** [comment]
We could reduce the intensity of the dim. I will go back and remove the formatting fixes on the PR (I am going to blame it on my auto linter).

File: static/sass/developer-include-topbar.scss [hunk 1]
```diff
@@ -325,18 +325,18 @@
...
 				}
 
 				&:hover {
 					+.overlay {
-						left: 0;
 						opacity: 1;
-						backdrop-filter: blur(6px);
-						transition: backdrop-filter ease-in-out 1s 0.1s;
+                        left: 0%;
+                        backdrop-filter: brightness(30%);
```
Comment (flynnduism) [inline static/sass/developer-include-topbar.scss:338]: Changing the `blur` to `brightness` is still problematic for browsers that struggle with the CSS `filter` effects - mostly Safari on iOS which will often crash and fail to load the page. 


**Accessibility / perf feedback:**
The `filter` and `backdrop-filter` classes should be avoided - you could do a less demanding `background-color: rgba(0,0,0,0.3);` (dark) or `background-color: rgba(255,255,255,0.3);` (light) approach and improve browser handling.

**Design feedback**
This :hover overlay effect is a nice-to-have flourish, and IMO it feels a bit heavy handed. It's a bit subjective, but a full page overlay is more typical with something like a modal than a dropdown menu. If it's taking over the page then ideally it's brief and subtle (not high contrast). 

The preview of this PR looks fine on dark mode, but is too high-contrast on light mode IMO.

**Comment by itowlson** [comment]
Incidentally, if addressing this is going to turn into a marathon, then I'm happy to leave it as is.  It's a cosmetic niggle and I only brought it up again because I was going through old issues seeing if any could be closed.

**Comment by karthik2804** [comment]
@itowlson  - It is a small change and I think it would be good to tackle in terms of browser compatibility and also website performance. 

Here is an updated preview link
https://fermyon-developer-twia6xje.fermyon.app/

cc: @flynnduism

**Comment by itowlson** [comment]
Thanks!  The dark mode now feels gentle and subtly muted.  However, the light mode seems to _brighten_ the page, rather than dimming it.  Is that intentional?  For me left it looking the page looking a bit washed out - subjective I know though!

**Comment by karthik2804** [comment]
@itowlson that was intentional as if not, we run again into the same contrast problem but I can take another stab at it.

**Review by flynnduism — APPROVED** [review]
I like how this edits looks and feels - subtle, doesn't distract or takeover the page as much.

There’s a slight a difference in the implementation between the light modes, but IMO it doesn’t matter, individually they both feel nice.

**Pull request merged**